levels/lovelace_tower/objectives/api-03-fetch/walkthrough.md

```diff
@@ -0,0 +1,63 @@
+# What is fetch?
+
+Sometimes we want to get data or perform an action on another computer. One common way to do this is by visiting websites. Whenever you visit a website, your computer sends a request. The request's configuration helps determine what happens when the other device receives it. Javascript provides the `fetch` method, which allows you to send such requests to other computers based on a [URL](https://developer.mozilla.org/en-US/docs/Web/API/URL) (Uniform Resource Locator) you provide.
+
+## Using fetch
+
+The `fetch` function takes two arguments, one is the URL that you're sending the request to and the other is the configuration. We'll be focussing on just a few of the configuration options here (the `method` option being among them), but there are [tons of others](https://developer.mozilla.org/en-US/docs/Web/API/fetch#parameters) to play around with. The `method` option is one of the most important to set when making requests, because it often drastically changes the response you get. You can set `method` to any of these [HTTP Verbs](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods), more on that coming up.
+
+## What are HTTP Verbs?
+
+HTTP Verbs (i.e. methods) represent the types of requests you can make. The most commonly used methods are `GET`, `POST`, `PATCH`, `PUT`, and `DELETE`.
+
+- `GET` asks for some sort of data or information from the other device
+- `POST` asks for something to be created by the other device
+- `PUT` asks for something that exists already to be replaced entirely by the other device with what's being sent in the request
+- `PATCH` asks for something that exists already to be updated by the other device using a set of instructions being sent in the request
+- `DELETE` asks for something to be deleted by the other device
+
```

Comment:
Might be worth mentioning that a response should still be sent back by all the requests, not just GET, and those responses can contain data just like GET